introduction.md

```diff
@@ -1 +1,5 @@
 Hi. I'm 7001. My favorite Code School Path is JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?